ui/app/actions.js

```diff
@@ -626,11 +626,19 @@ function markAccountsFound() {
 //
 
 function setRpcTarget (newRpc) {
-  if (global.METAMASK_DEBUG) console.log(`background.setRpcTarget`)
-  background.setRpcTarget(newRpc)
-  return {
-    type: actions.SET_RPC_TARGET,
-    value: newRpc,
+  return (dispatch) => {
+    if (global.METAMASK_DEBUG) console.log(`background.setRpcTarget`)
```

Comment:
We now have a method `log.debug(message)`, which automatically conditionalizes.  There are even different log levels.  You can learn more on npm under `loglevel`.